src/plugins/intel_cpu/tools/commit_slider/commit_slider.py

```diff
@@ -7,7 +10,11 @@
 
 args, cfgData, customCfgPath = getParams()
 
-if args.__dict__["isWorkingDir"]:
+if args.__dict__["utility"] != "no_utility":
```

Comment:
minor: can we use `args.utility` and the same approach for other parameters instead `args.__dict__["utility"]`?

Reply:
Applied